food-truck-frontend/src/components/util/ImageDialog.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import {Button, Dialog, DialogContent, DialogTitle, Grid, IconButton} from "@material-ui/core";
+import {Close} from "@material-ui/icons";
+
+interface ImageDialogProps {
+  url: string;
+  text: string;
+}
+
+function ImageDialog(props: ImageDialogProps) {
+  const [open, setOpen]: [boolean, any] = useState(false);
+
+  return (
+    <>
+      <Button color="primary"
+                  onClick={() => setOpen(true)}>
+        View {props.text}
+      </Button>
+      <Dialog open={open}
+              fullWidth
+              maxWidth="md">
+        <Grid container direction="row" justify="flex-start">
```

Comment:
I'd make the same comment about how Dialogs work as before, but this looks good UI-wise, so it's fine to leave it alone.